### ISSUE TITLE
MS SQL Server Support

### ISSUE THREAD

**Issue by MiConnell** [akdor1154/python-csql#4]
SQL Server doesn't use the `LIMIT` keyword but uses `TOP` instead

`SELECT TOP 10 * FROM CUSTOMERS`

instead of

`SELECT * FROM CUSTOMERS LIMIT 10`

I'd be happy to work on a PR for this.

**Comment by akdor1154** [comment]
Ah bugger, yeah you're right. Damn, that'd be a very early end for my hope for no sql dialect stuff.
Yeah I'd accept a PR, for now I reckon just add a kwarg to preview_pd with dialect='mssql'.

At one point I was using chunksize=n on read_sql instead of wrapping in a `limit` query, but that failed for queries with no results; if that could be made to work for empty results I'd prefer that, but it probably requires a pandas-side fix.

**Comment by MiConnell** [comment]
Actually could you just filter the DataFrame instead? There might be SQL-side performance issues if the select statement pulls from millions of rows, but that way it could still work regardless of dialect.
Something like

```python
previewQ = Q(lambda: f"""select * from {self}""", p)
return pd.read_sql(**previewQ.build().pd, con=con).head(rows)
```

**Comment by akdor1154** [comment]
No sorry, I would like preview to limit rows pulled by default.

**Comment by akdor1154** [comment]
The pandas issue might get fixed soon, see https://github.com/pandas-dev/pandas/issues/34411

**Comment by MiConnell** [comment]
Ok well if you'd like I should have some time this week to work on this.

**Comment by akdor1154** [comment]
this should be fixed in 0.7.0: set `csql.dialect.MSSQL` as your query dialect.
